components/cosmetic_filters/resources/data/element_picker.ts

```diff
@@ -593,6 +593,21 @@ const launchElementPicker = (root: ShadowRoot) => {
   if (!isAndroid) {
     section.classList.add('desktop')
   }
+
+  if (isAndroid) {
+    const mediaQuery = window.matchMedia("(orientation: landscape)");
```

Comment:
```
Deprecated: This feature is no longer recommended. Though some browsers might still support it, it may have already been removed from the relevant web standards, may be in the process of being dropped, or may only be kept for compatibility purposes. Avoid using it, and update existing code if possible; see the [compatibility table](https://developer.mozilla.org/en-US/docs/Web/API/Window/orientation#browser_compatibility) at the bottom of this page to guide your decision. Be aware that this feature may cease to work at any time.
```
WDYT about https://developer.mozilla.org/en-US/docs/Web/API/Screen/orientation?